### PR TITLE
added the base project code

### PR DESCRIPTION
**Pull request by ionisea**

### CODE REVIEW

File: projects/physics/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "type": "module",
```
Comment (szetingchen) [inline projects/physics/package.json:2]: bad code

**Review by szetingchen — APPROVED** [review]
actually reviewed

File: projects/physics/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
+ * Trivial web server that serves the contents of the public/ directory.
+ */
+
+import express from 'express';
+
+const port = 3000
```
Comment (lashlockbhs) [inline projects/physics/index.js:7]: awesome

**Review by lashlockbhs — APPROVED** [review]
lgtm

**Pull request merged**